include/z64vis.h

```diff
@@ -0,0 +1,87 @@
+#ifndef Z64_VIS_H
+#define Z64_VIS_H
+
+#include "ultra64.h"
+#include "color.h"
+
+typedef enum {
+    /* 0 */ FB_FILTER_NONE,
+    /* 1 */ FB_FILTER_CVG_RGB,
+    /* 2 */ FB_FILTER_CVG_RGB_UNIFORM,
+    /* 3 */ FB_FILTER_CVG_ONLY,
+    /* 4 */ FB_FILTER_CVG_RGB_FOG, // Not recommended, easily overflows blender
+    /* 5 */ FB_FILTER_ZBUF_IA,
+    /* 6 */ FB_FILTER_ZBUF_RGBA,
+    /* 7 */ FB_FILTER_MONO
+} FramebufferFilterType;
+
+typedef enum {
+    /* 0 */ VIS_NO_SETSCISSOR,
+    /* 1 */ VIS_SETSCISSOR
+} VisScissorType;
+
+typedef struct {
+    /* 0x00 */ u32 type;
+    /* 0x04 */ u32 scissorType;
+    /* 0x08 */ Color_RGBA8_u32 primColor;
+    /* 0x0C */ Color_RGBA8_u32 envColor;
+} Vis; // size = 0x10
```

Comment:
Could you put the name at the top of the struct/enums too?
```suggestion
typedef enum FramebufferFilterType {
    /* 0 */ FB_FILTER_NONE,
    /* 1 */ FB_FILTER_CVG_RGB,
    /* 2 */ FB_FILTER_CVG_RGB_UNIFORM,
    /* 3 */ FB_FILTER_CVG_ONLY,
    /* 4 */ FB_FILTER_CVG_RGB_FOG, // Not recommended, easily overflows blender
    /* 5 */ FB_FILTER_ZBUF_IA,
    /* 6 */ FB_FILTER_ZBUF_RGBA,
    /* 7 */ FB_FILTER_MONO
} FramebufferFilterType;

typedef enum VisScissorType {
    /* 0 */ VIS_NO_SETSCISSOR,
    /* 1 */ VIS_SETSCISSOR
} VisScissorType;

typedef struct Vis {
    /* 0x00 */ u32 type;
    /* 0x04 */ u32 scissorType;
    /* 0x08 */ Color_RGBA8_u32 primColor;
    /* 0x0C */ Color_RGBA8_u32 envColor;
} Vis; // size = 0x10
```